sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/OpenCoverReportParserTest.java

```diff
@@ -189,10 +189,11 @@ public void branchCoverage_getter_setter_multiple_sequence_points_per_line() thr
       List<String> traceLogs = logTester.logs(LoggerLevel.TRACE);
       assertThat(traceLogs).hasSize(35);
 
-      String logLine = traceLogs.get(32);
-      assertThat(logLine)
-        .startsWith("Found coverage information about '10' lines having single-line sequence points for file '")
-        .endsWith("Bar.cs'");
+
+      assertThat(traceLogs)
```

Comment:
I've modified this since the order changed. I've made it now to not depend on order.